TicTacToe/TicTacToe.py

```diff
@@ -0,0 +1,129 @@
+def get_board_size():
```

Comment:
the file tictactoe should not be here, this is the pull request for list sum. I think this is because you did not use two seperate branches appropriatly.